.env.example

```diff
@@ -52,6 +52,8 @@ POSTGRES_DB=parabol-saas
 POSTGRES_HOST=localhost
 POSTGRES_PORT=5432
 PROTO='http'
+# RATE_LIMITER: in-memory | stub
+RATE_LIMITER='stub'
```

Comment:
See https://github.com/ParabolInc/action-devops/pull/83 for the updates to the environment files.